<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsVectorLayerSaveAsDialogBase</class>
 <widget class="QDialog" name="QgsVectorLayerSaveAsDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>632</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Save Vector Layer asâ€¦</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="3" column="0">
       <widget class="QLabel" name="mCrsLabel">
        <property name="text">
         <string>CRS</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QgsFileWidget" name="mFilename" native="true">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Format</string>
        </property>
        <property name="buddy">
         <cstring>mFormatComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="mFormatComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>File name</string>
        </property>
        <property name="buddy">
         <cstring>mFilename</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QgsProjectionSelectionWidget" name="mCrsSelector" native="true">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Layer name</string>
        </property>
        <property name="buddy">
         <cstring>leLayername</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QgsFilterLineEdit" name="leLayername">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QgsScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>557</width>
        <height>1000</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Encoding</string>
           </property>
           <property name="buddy">
            <cstring>mEncodingComboBox</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="mEncodingComboBox"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="mSelectedOnly">
         <property name="text">
          <string>Save only selected features</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QgsCollapsibleGroupBox" name="mAttributesSelection">
         <property name="title">
          <string>Select fields to export and their export options</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="collapsed" stdset="0">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QTableWidget" name="mAttributeTable">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>200</height>
             </size>
            </property>
            <property name="rowCount">
             <number>1</number>
            </property>
            <property name="columnCount">
             <number>2</number>
            </property>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <row/>
            <column/>
            <column/>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPushButton" name="mSelectAllAttributes">
              <property name="text">
               <string>Select All</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="mDeselectAllAttributes">
              <property name="text">
               <string>Deselect All</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="mUseAliasesForExportedName">
            <property name="text">
             <string>Use aliases for exported name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="mReplaceRawFieldValues">
            <property name="text">
             <string>Replace all selected raw field values by displayed values</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="mCheckPersistMetadata">
         <property name="toolTip">
          <string>If checked, any metadata present in the source layer will be copied to the output layer.</string>
         </property>
         <property name="text">
          <string>Persist layer metadata</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="mSymbologyExportLabel">
           <property name="text">
            <string>Symbology export</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="mSymbologyExportComboBox"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="mScaleLabel">
           <property name="text">
            <string>Scale</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QgsScaleWidget" name="mScaleWidget" native="true">
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QgsCollapsibleGroupBox" name="mGeometryGroupBox">
         <property name="title">
          <string>Geometry</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="mSymbologyExportLabel_2">
              <property name="text">
               <string>Geometry type</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="mGeometryTypeComboBox"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="mIncludeZCheckBox">
            <property name="text">
             <string>Include z-dimension</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="mForceMultiCheckBox">
            <property name="text">
             <string>Force multi-type</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QgsExtentGroupBox" name="mExtentGroupBox">
         <property name="title">
          <string>Extent</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QgsCollapsibleGroupBox" name="mDatasourceOptionsGroupBox">
         <property name="title">
          <string>Datasource Options</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QgsCollapsibleGroupBox" name="mLayerOptionsGroupBox">
         <property name="title">
          <string>Layer Options</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QgsCollapsibleGroupBox" name="mOgrOptionsGroupBox">
         <property name="title">
          <string>Custom Options</string>
         </property>
         <property name="collapsed" stdset="0">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Data source</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <property name="buddy">
             <cstring>mOgrDatasourceOptions</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QTextEdit" name="mOgrDatasourceOptions">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Layer</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <property name="buddy">
             <cstring>mOgrLayerOptions</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QTextEdit" name="mOgrLayerOptions">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
         <zorder>label_6</zorder>
         <zorder>label_5</zorder>
         <zorder>mOgrDatasourceOptions</zorder>
         <zorder>mOgrLayerOptions</zorder>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout1">
     <item>
      <widget class="QDialogButtonBox" name="mHelpButtonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Help</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="mAddToCanvas">
       <property name="text">
        <string>Add saved file to map</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="mButtonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsScrollArea</class>
   <extends>QScrollArea</extends>
   <header>qgsscrollarea.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsProjectionSelectionWidget</class>
   <extends>QWidget</extends>
   <header>qgsprojectionselectionwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsExtentGroupBox</class>
   <extends>QgsCollapsibleGroupBox</extends>
   <header>qgsextentgroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsScaleWidget</class>
   <extends>QWidget</extends>
   <header>qgsscalewidget.h</header>
  </customwidget>
  <customwidget>
   <class>QgsFilterLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qgsfilterlineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mFormatComboBox</tabstop>
  <tabstop>leLayername</tabstop>
  <tabstop>mCrsSelector</tabstop>
  <tabstop>scrollArea</tabstop>
  <tabstop>mEncodingComboBox</tabstop>
  <tabstop>mSelectedOnly</tabstop>
  <tabstop>mAttributeTable</tabstop>
  <tabstop>mSelectAllAttributes</tabstop>
  <tabstop>mDeselectAllAttributes</tabstop>
  <tabstop>mReplaceRawFieldValues</tabstop>
  <tabstop>mCheckPersistMetadata</tabstop>
  <tabstop>mSymbologyExportComboBox</tabstop>
  <tabstop>mScaleWidget</tabstop>
  <tabstop>mGeometryTypeComboBox</tabstop>
  <tabstop>mForceMultiCheckBox</tabstop>
  <tabstop>mIncludeZCheckBox</tabstop>
  <tabstop>mOgrDatasourceOptions</tabstop>
  <tabstop>mOgrLayerOptions</tabstop>
  <tabstop>mAddToCanvas</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
